ui/components/app/terms-of-use-popup/index.scss

```diff
@@ -0,0 +1,35 @@
+.popover-wrap.terms-of-use__popover {
+  .terms-of-use {
+    &__content {
+      display: flex;
+      flex-direction: column;
+    }
+
+    &__terms-list {
+      list-style: decimal none outside;
+    }
+
+    &__footer-text {
+      align-self: center;
+    }
+  }
+
+  .popover-header {
+    &__title {
+      margin-bottom: 0;
+    }
+  }
+
+  .popover-footer {
+    border-top: none;
+  }
+
+  @include screen-sm-min {
+    max-height: 750px;
+    width: 500px;
```

Comment:
Why are we setting the width to 500px here?

Reply:
I think we wanted the dimensions to fully cover the "What's New" popover since it displays on top of it, so I just used the same stylings we had for the what's new popover